bin/diagnostic.js

```diff
@@ -1 +1,16 @@
 'use strict'
+
+const ask = require('../lib/ask.js')
+
+let answer = 'not empty'
+
+while (answer !== '') {
+  answer = ask("Guess my secret? ")
```

Comment:
Looks good, but you want to give the user a specific number to guess in addition to making sure their answer falls within the range